tests/format-quick-bar.spec.ts

```diff
@@ -35,7 +35,7 @@ test('should format quick bar show when select text', async ({ page }) => {
     throw new Error("formatQuickBar doesn't exist");
   }
   assertAlmostEqual(box.x, 20, 5);
-  assertAlmostEqual(box.y, 260, 5);
```

Comment:
Need to conform with @lawvs. What's these number should be?

Reply:
This update is ok